_posts/2018/2018-01-07-be-sparing-when-using-unless.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Be sparing when using unless'
+description: 'Gimme some sugar, but not too much'
+layout: article
+category: ruby
+image:
+  base: '2018/be-sparing-when-using-unless'
+  alt: 'Two directions sign'
+  source: 'https://unsplash.com/photos/lPQIndZz8Mo'
+  credit: 'Pablo Garcia Saldaña'
+---
+
+One of Ruby’s strengths is its delightful, built-in syntactic sugar. One example of this is the keyword `unless`, which you can use in place of using `if` combined with a negative statement.
+
+However, given the flexibility of Ruby's syntax, it is easy to make code harder to understand than it needs to be.
+
+## Instead of…
+
+...using `unless` in ways that makes your brain hurt.
+
+```ruby
+# Example 1
+unless something?
```

Comment:
I think you want to remove the `?` here, or add one on line 42, for continuity purposes?

Reply:
Added on 42.